### ISSUE TITLE
v2 issue only- GBIF

### ISSUE THREAD

**Issue by ecurrano** [paleobot/pbot-dev#293]
Recording this here so that we don't lose it. This is NOT a v1 concern.

Based on the workshop I attended last week, it seems that iDigBio is being phased out and the specimen database du jour is GBIF. (Holly Little, Erica Krimmel, and Talia Karim led the workshop.) However, if I understood correctly, iDigBio is sending things to GBIF. Anyway, we should probably stop writing about integration with iDigBio and instead think about integration with GBIF for the future.

**Comment by aazaff** [comment]
Should iDigBio be removed from the specimens forms and nodes? If I recall, there was something about the GBIF API that made it difficult for us to use which is why we ended up dropping it from the agenda back when we added iDigBio. (I vaguely recall that they did not have a consistent UUID format)... anyway... if we're ditching idigbio maybe better to do sooner than later so we don't have to clean up a bunch of legacy data later?

**Comment by doricon** [comment]
yes, Andrew you are correct, our issue with integrating with GBIF is there is not a direct link to use like there is with iDigBio. Basically GBIF sends you to the original data provider. 

I don't think we should do anything about this for now, because the idigbio links aren't going to disappear, to my understanding....
